ed25519.py

```diff
@@ -0,0 +1,151 @@
+import hashlib
```

Comment:
I don't think we need to include this file: linking to it in a comment is probably fine.